apps/molgenis-components/src/components/layout/SimpleModal.vue

```diff
@@ -69,3 +70,38 @@ article {
   border-radius: 4px;
 }
 </style>
+
+<docs>
```

Comment:
I suppose we also should try to merge this one with the LayoutModal? And make sure LayoutModal can also do this?